device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/gearbox_config.json

```diff
@@ -0,0 +1,334 @@
+{
+  "phys": [
+    {
+      "phy_id": 1,
+      "name": "phy1",
+      "address": "1",
+      "lib_name": "",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy1_config_1.json",
+      "sai_init_config_file": "",
+      "phy_access": "mdio",
+      "bus_id": 0,
+      "context_id": 1,
+      "hwinfo": "mdio0_0_0/0"
+    },
+    {
+      "phy_id": 2,
+      "name": "phy2",
+      "address": "2",
+      "lib_name": "",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy2_config_1.json",
+      "sai_init_config_file": "",
+      "phy_access": "mdio",
+      "bus_id": 0,
+      "context_id": 1,
+      "hwinfo": "mdio1_0_0/0"
+    },
+    {
+      "phy_id": 3,
+      "name": "phy3",
+      "address": "3",
+      "lib_name": "",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy3_config_1.json",
+      "sai_init_config_file": "",
+      "phy_access": "mdio",
+      "bus_id": 0,
+      "context_id": 1,
+      "hwinfo": "mdio2_0_0/0"
+    },
+    {
+      "phy_id": 4,
+      "name": "phy4",
+      "address": "4",
+      "lib_name": "",
+      "firmware_path": "/usr/share/phy_fw.bin",
+      "config_file": "/usr/share/sonic/hwsku/phy4_config_1.json",
```

Comment:
>_1 [](http://example.com/codeflow?start=56&length=2)

is there a reason to add _1 here? seems not needed.

Reply:
We followed "src/sonic-swss/gearsyncd/tests/configs/positive/phy1_config_1.json".  Other than that, it is not needed.